### PR TITLE
[DS-5229][feat] Implement server port custom config

### LINKED ISSUE #5229: [Feature][server port] server port custom config
**Describe the feature**
all application can custom config server port

### PR DESCRIPTION
**Pull request by Lucaszlei**
[DS-5229][feat] Implement server port custom config
This closes #5229

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ private Constants() {
     /**
      * alter properties
      */
-    public static final int ALERT_RPC_PORT = 50052;
+    public static final String ALERT_RPC_PORT = "alter.rpc.port";
```
Comment (davidzollo) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: alter.rpc.port? is it `alert.rpc.port`?

File: dolphinscheduler-common/src/main/resources/common.properties [hunk 2]
```diff
@@ -89,3 +89,7 @@ sudo.enable=true
 
 # development state
 development.state=false
+
+# rpc port
+rpc.port=50051
+alter.rpc.port=50052
```
Comment (davidzollo) [inline dolphinscheduler-common/src/main/resources/common.properties]: alert, not alter

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java (hunk 1 above)
Comment (Lucaszlei) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: sorry，I revised it

The word "alert" in the comment is also wrong. Did I modify it by the way? Or amend it through a new issue?

Comment (Lucaszlei) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: > alter.rpc.port? is it `alert.rpc.port`?

@dailidong

Comment (davidzollo) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: you can modify it in this PR

Comment (davidzollo) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: please modify it in this PR, thx

Comment (Lucaszlei) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: > please modify it in this PR, thx

Modified to complete

Comment (Lucaszlei) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java]: @dailidong What to do next?

**Comment by kezhenxu94** [comment]
It is better to put the config item in https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/resources/application-alert.yaml

**Comment by davidzollo** [comment]
> It is better to put the config item in https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/resources/application-alert.yaml

yes, you can submit another PR

all in all, thanks for your first contribution, it's a good beginning, good luck

**Review by davidzollo — APPROVED** [review]
+1
good job

**Pull request merged**